plutus-tx/src/PlutusTx/Foldable.hs

```diff
@@ -55,6 +55,16 @@ import PlutusTx.Semigroup ((<>))
 
 import Prelude qualified as Haskell (Monad, return, (>>), (>>=))
 
+{- Note [newtype field accessors in `base`]
```

Comment:
Maybe this should go in the style guide too? Or somewhere more visible.

Reply:
I don't know about style guide since it's not exact a style thing and it also only applies to code compiled by the plugin, but I agree this module is not the best place for it, and I moved it to Plugin.hs.